docs/DafnyRef/UserGuide.md

```diff
@@ -240,7 +240,7 @@ Various options control the translation process, in addition to all those descri
 
 - The translation results
    - `--output` (or `-o`) --- location (file or folder depending on the target) of the generated file(s)
-   - `--include-runtime` --- include the Dafny runtime as source or a library in the target language
+   - `--include-runtime` --- include the Dafny runtime as source or a library for the target language in the generated artifacts
```

Comment:
What does `or a library` mean here? 

This works, but consider saying `include the Dafny runtime as target language source in the generated artifacts`